clients/imodels-client-management/src/operations/briefcase/BriefcaseOperationParams.ts

```diff
@@ -4,18 +4,29 @@
  *--------------------------------------------------------------------------------------------*/
 import { CollectionRequestParams, IModelScopedOperationParams } from "../../base";
 
+/** Special value which is an alias for the current user id. */
```

Comment:
Special value to indicate the id of current user who is making the request

Reply:
Fixed: https://github.com/iTwin/imodels-clients/blob/b3a0ba5c80c6c55204fa675ed18de31a703c9e36/clients/imodels-client-management/src/operations/briefcase/BriefcaseOperationParams.ts#L7